addons/website_sale_picking/models/website.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    picking_site_ids = fields.Many2many(
+        'delivery.carrier',
+        string='Picking sites'
+    )
```

Comment:
Do what you want, it's only a suggestion :) 

```suggestion
    picking_site_ids = fields.Many2many('delivery.carrier', string='Picking sites')
```